sourcemash/api/feeds.py

```diff
@@ -18,14 +20,75 @@
 
 class FeedListAPI(Resource):
```

Comment:
Should there be a delete to actually remove a feed from the database? It's not something that user's should be able to do, but maybe admins? For example, if an admin wanted to remove an "offensive" feed?